config/environments/development.rb

```diff
@@ -50,6 +50,15 @@
     path: "/:class/:attachment/:id_partition/:style/:filename"
   }
 
+  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.sendmail_settings = {
+    location: ENV['SENDMAIL_PATH'],
+    arguments: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.